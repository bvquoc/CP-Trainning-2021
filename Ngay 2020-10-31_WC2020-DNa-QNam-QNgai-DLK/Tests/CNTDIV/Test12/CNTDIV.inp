50
3425
3744
3051
2019
1783
1806
4081
1913
9719
4218
1956
1120
4790
5327
2218
6219
4034
2263
1728
1674
1787
1799
7750
1274
3857
1873
2678
4041
2265
1244
1508
2773
1374
1511
3014
4873
1533
2017
1662
3594
3071
3430
1213
5409
5107
1130
1188
5522
4011
3676
