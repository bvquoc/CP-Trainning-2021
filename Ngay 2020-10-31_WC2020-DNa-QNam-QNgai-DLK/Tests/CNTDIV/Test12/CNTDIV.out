1631
7226
3869
1631
2296
3626
3025
966
848
2296
2296
13320
934
2678
2296
869
723
2296
11734
2924
155
2075
21684
20111
3025
285
9202
1631
966
2296
2296
5084
3238
1729
15443
2296
3025
90
545
2296
2200
21684
350
1631
512
2296
16837
2296
1631
220
